frontend/src/bundles/talent-onboarding/enums/steps.enum.ts

```diff
@@ -2,7 +2,7 @@ const Steps = {
     STEP_01: 'Profile',
```

Comment:
```js
    Step.STEP_1: 'Profile',
```
you can reuse enum here 
https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/blob/b84788183c2c9e4e428a8a75469d2dfbb03695ed/shared/src/bundles/user-details/enums/completed-step.enum.ts